pages/guide/08-start.md

```diff
@@ -0,0 +1,422 @@
+    },
+    {
+       name: coachState
+       to: country
+       kind: toOne
+       type: right
+       definition: '{{state_id}} = {{coachState.id}}'
+    },
+    {
+       name: playerTeam
+       to: team
+       kind: toMany
+       type: full
+       definition: '{{team_id}} = {{playerTeam.id}}'
+    }
+]
+
+```
+
+For more information on joins, please check out: [https://elide.io/pages/guide/v5/04-analytics.html#joins](https://elide.io/pages/guide/v5/04-analytics.html#joins)
+
+### How do I union tables together in my semantic config?
+
+If you want to UNION the data from 2 or more tables and present as a single unified table to the user, you can provide a SQL subquery in the Hjson config.
+
+```
+{
+  tables: [{
+      name: PlayerStats
+      sql: '''
+      select name, game_type, created_on, highScore from playerStatsPractice UNION ALL select name, game_type, created_on, highScore from playerStatsChampionship
+      '''
+      dbConnectionName: Presto Data Source
+      friendlyName: Player Stats
+      description:
+      '''
+      A long description
+      '''
+      category: Sports
+      cardinality : large
+      readAccess : '(user AND member) OR (admin.user AND NOT guest user)'
+      isFact : true
+      tags: ['Game', 'Player']
+      measures : [
+          {
+          name : highScore
+          type : INTEGER
+          definition: 'MAX({{highScore}})'
+          friendlyName: High Score
+          }
+      ]
+      dimensions : [
+          {
+              name : name
+              type : TEXT
+              definition : '{{name}}'
+              cardinality : large
+          },
+          {
+              name : gameType
+              type : TEXT
+              definition : '{{game_type}}'
+              friendlyName: Game Type
+          },
+          {
+              name : createdOn
+              type : TIME
+              definition : '{{created_on}}'
+              grain:
+              {
+                  type :  DAY
+                  sql :  '''PARSEDATETIME(FORMATDATETIME({{}}, 'yyyy-MM-dd'), 'yyyy-MM-dd')'''
+              }
+          }
+      ]
+  }]
+}
+```
+
+For more information on subqueries please check out: [https://elide.io/pages/guide/v5/04-analytics.html#table-properties](https://elide.io/pages/guide/v5/04-analytics.html#table-properties)
+
+### How to i convert data of one type into another type in my semantic config?
+
+If you want to convert data from one type to another, you can provide a SQL definition for the column in the Hjson config. See below for an example of converting a Timestamp to String in 'YYYY-MM-DD' format.
+
+```
+{
+    name : createdOn
+    type : TEXT
+    definition : '''
+    FORMATDATETIME({{created_on}},'yyyy-MM-dd')
+    '''
+}
+```
+
+For more information on SQL definition please check out: [https://elide.io/pages/guide/v5/04-analytics.html#analytic-queries](https://elide.io/pages/guide/v5/04-analytics.html#analytic-queries)
```

Comment:
We should link to https://elide.io/pages/guide/v5/04-analytics.html#column-properties